pkg/apis/core/v1alpha1/federationconfig_types.go

```diff
@@ -17,13 +17,15 @@ limitations under the License.
 package v1alpha1
 
 import (
+	apiextv1b1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // FederationConfigSpec defines the desired state of FederationConfig
 type FederationConfigSpec struct {
-	// Whether the federation namespace will be the only target for federation.
-	LimitedScope bool `json:"limited-scope,omitempty"`
+	// The scope of the federation control plane should be either `Namespaced` or `Cluster`
```

Comment:
Please end this line with a period, since the entirety of the comment will be added together in the crd field doc.

Reply:
@marun done